### ISSUE TITLE
The version of torch,qiskit,qiskit_machine_learning

### ISSUE THREAD

**Issue by CaiJichang212** [contepablod/QCNNCancerClassifier#1]
Thank you very much for open sourcing the QCNN code.
Since your code was open sourced 2 years ago, in order to run your code correctly I need to know the versions of torch,qiskit,qiskit_machine_learning which are the 3 core 3rd party libraries.
I hope you can provide it. Thank you very much.

**Comment by contepablod** [comment]
Yes, it has some time. I should refactor it with several stuff regarding torch and qiskit. Follow the repo and when it is done you will see the update. Thanks for your reach.
